cmd/grafana-reporter/main.go

```diff
@@ -92,6 +93,17 @@ func apiToken(r *http.Request) string {
 	return apiToken
 }
 
+func dashVariable(r *http.Request) string {
+	if strings.Contains(r.URL.RequestURI(), "&var-") == true {
```

Comment:
This won't match `?var-foo=bar`

Reply:
Thanks for spotting that. I'll see if I can add some tests when I merge this.